code-pushup.config.ts

```diff
@@ -77,7 +78,9 @@ const config: CoreConfig = {
       type: 'module',
     }),
 
-    await lighthousePlugin('https://codepushup.dev/'),
+    await lighthousePlugin('https://codepushup.dev/', {
+      chromeFlags: DEFAULT_FLAGS.concat(['--headless']),
```

Comment:
Should we set these Chrome flags by default and make `chrome-launcher` a dependency of `@code-pushup/lighthouse-plugin`? :thinking:

Reply:
Thought about it and decided against listing it as dependency. 
But is should be indirectly here anyway... https://github.com/GoogleChrome/lighthouse/blob/369979f498bd6560127e10476edffb264d4fa3b9/cli/run.js#L13

Reply:
What's the argument for not making these flags the default? Based on your description of "more stables runs", it sounds like something I would want on every project.

Reply:
I also believe it, I am just a bit unsure and wanted to have it in use for a while. 
I could surely move it into the plugin.